frontend/src/features/public-form/components/FormFields/FormFields.tsx

```diff
@@ -163,19 +163,20 @@ export const FormFields = ({
             </Stack>
           </Box>
         )}
-        {form?.responseMode === FormResponseMode.Encrypt && (
-          <Box
-            mt="2.5rem"
-            bg="white"
-            py="2.5rem"
-            px={{ base: '1rem', md: '2.5rem' }}
-          >
-            <PaymentPreview
-              colorTheme={colorTheme}
-              paymentDetails={form?.payments_field}
-            />
-          </Box>
-        )}
+        {form?.responseMode === FormResponseMode.Encrypt &&
+          form?.payments_field.enabled && (
```

Comment:
Just a nit: I think as we moved the validation of payment details being enabled above the box, we can now confirm that the `paymentDetails` thats being passed into `PaymentPreview` is defined. 

That can also help to remove the 

`if (!paymentDetails || !paymentDetails.enabled) return null` check within `PaymentPreview` too

Reply:
done in [e896b37](https://github.com/opengovsg/FormSG/pull/6342/commits/e896b37be6c91de0c872ea34d356e140b1a416a7)